### DISCUSSION TITLE
how to disable checking for update

### DISCUSSION THREAD

**Discussion by DirektX** [Rem0o/FanControl.Releases#1065] [Ideas]
We need an option to disable checking for update.
Thank you

**Comment by Rem0o** [comment]
There is already. When the update pop-up comes up, there is a checkbox for that.

**Comment by DirektX** [comment]
OK, but that checkbox is only for the actual version.
But more important is this: when Fan Control wants to check, my firewall kicks in _before_ the popup. So I need an option to disable _any_ update check at _any_ time.

**Reply by Rem0o** [reply]
Ahhhh, the checkbox right now disable the popup, but the option is only applied after the server call to check for the current version is made, which is unnecessary. 

Will fix that.

**Reply by Rem0o** [reply]
V126 fixes this.

**Reply by DirektX** [reply]
Please be gentle for me.
I still can not find any option to disable updating.

**Reply by Rem0o** [reply]
When the updater pop-up appears next time, check the checkbox. After that, it won't trigger your firewall every boot.
